src/utils/Processing.ts

```diff
@@ -240,6 +241,29 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
+  for (const property in resource) {
+    // We'll want to exclude properties beginning with an "_" as those are arrays containing extensions
+    // corresponding to a primitive array, per the FHIR spec
+    if (property.startsWith('_')) continue;
+    if (_.isPlainObject(resource[property]))
```

Comment:
I think the convention we've been using is to import just the functions we need rather than all of lodash.  So I'd change the import to:
```
import { isPlainObject } from 'lodash';
```
and just use `isPlainObject` here.